docker/worker/README.md

```diff
@@ -0,0 +1,25 @@
+## Docker configuration file for the Turbinia worker
+This Docker configuration is for building Turbinia worker and server images to be run on GCP (with Terraform) or in a Hybrid local/GCP setup.
+
+### Build the image
+```
+docker build -t turbinia-worker:dev .
+```
+
+### Base64 encode turbinia.conf
+```
+base64 -w 0 turbinia.conf > turbinia.conf.b64
+```
+-w 0 will output without column wrapping (not needed on OSX).
+
+### Hybrid GCP - setup credentials
+See https://cloud.google.com/docs/authentication/getting-started and download the credentials to ./gcp/creds.json
+
+### Run the worker
+```
+docker run -ti \
+-v $PWD/gcp:/home/turbinia/gcp \
```

Comment:
We don't need to map in anything in /dev?

Reply:
Nope, mapping is very specific to GCP and it makes no sense to map /dev (as a whole) from a non-GCP host system to the container. When running outside of GCP loop devices are used afaik and they do not require mapping.